src/views/CustomerStory.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="home">
+    <nav
+      class="navbar is-link is-fixed-top"
+      role="navigation"
+      aria-label="main-navigation"
+    >
+      <div class="navbar-brand">
+        <a
+          role="button"
+          class="navbar-burger burger"
+          aria-label="menu"
+          aria-expanded="false"
+          data-target="navbar-menu"
+          :class="{ 'is-active': showNav }"
+          @click="toggleNav()"
+        >
+          <span aria-hidden="true"></span> <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+      <div
+        id="navbar-menu"
+        class="navbar-menu"
+        :class="{ 'is-active': showNav }"
+        @click="toggleNav()"
+      >
+        <div class="navbar-end">
+          <router-link to="/" class="navbar-item" href="#">Home</router-link>
+          <LocaleChanger :langs="{ en: 'English', de: 'Deutsch' }" />
+        </div>
+      </div>
+    </nav>
+
+    <br />
+    <br />
+    <section class="section">
+      <div class="columns is-centered columns-section">
+        <div class="column is-half">
+          <h2 class="title is-2 is-spaced">
+            {{ customer.title }}
+          </h2>
+          <figure class="image customer-story-logo">
+            <img :src="logo" :alt="customer.logo.alt" />
+          </figure>
+          <br />
+          <p>
+            <i>"{{ customer.quote.text }}"</i>
+          </p>
+          <br />
+          <p style="float: right">
+            {{ customer.quote.stakeholder.name }},
+            {{ customer.quote.stakeholder.role }}
+          </p>
+          <br />
+          <br />
+          <h3 class="title is-3">{{ $t("customer_story.customer") }}</h3>
+          <p>
+            <span v-html="customer.customer"></span>
```

Comment:
Please add `<!-- eslint-disable-next-line vue/no-v-html -->` to the line before to get rid of the linting warnings.

Reply:
Done.